roles/container-engine/docker/vars/debian.yml

```diff
@@ -12,14 +12,14 @@ docker_versioned_pkg:
   '18.03': docker-ce=18.03.1~ce-0~debian
   '18.06': docker-ce=18.06.2~ce~3-0~debian
   '18.09': docker-ce=5:18.09.9~3-0~debian-{{ ansible_distribution_release|lower }}
-  '19.03': docker-ce=5:19.03.11~3-0~debian-{{ ansible_distribution_release|lower }}
-  'stable': docker-ce=5:19.03.11~3-0~debian-{{ ansible_distribution_release|lower }}
-  'edge': docker-ce=5:19.03.11~3-0~debian-{{ ansible_distribution_release|lower }}
+  '19.03': docker-ce=5:19.03.12~3-0~debian-{{ ansible_distribution_release|lower }}
```

Comment:
Shouldn't the PR title be `Update docker package to 19.03.12` instead?
I checked the previous commit(953bc8dee2c60cb7ff07b3a5150cf05d9b57b0d7) which updated this part, the title is `19.03.11` as the same as the change.

Reply:
Oups thx man will update pr and commit!